### PR TITLE
Upgrade Symfony to v6 and EasyAdmin to v4

### PR DESCRIPTION
**Pull request by bocharsky-bw**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,47 +2,53 @@
...
-        "php": "^7.4.1 || ^8.0.0",
+        "php": ">=8.0.2",
         "ext-ctype": "*",
         "ext-iconv": "*",
         "composer/package-versions-deprecated": "^1.11",
         "doctrine/doctrine-bundle": "^2.1",
         "doctrine/doctrine-migrations-bundle": "^3.0",
         "doctrine/orm": "^2.7",
-        "knplabs/knp-markdown-bundle": "^1.8",
+        "knplabs/knp-markdown-bundle": "dev-symfony6",
```
Comment (weaverryan) [inline composer.json:12]: Nice job - I assume you created a PR for this to the bundle?

File: docker-compose.yaml [hunk 2]
```diff
@@ -10,3 +10,22 @@ services:
             # "3306" to "3306:3306". Where the first port is exposed to the host and the second is the container port.
             # See https://docs.docker.com/compose/compose-file/#ports for more information.
             - '3306'
+
+###> doctrine/doctrine-bundle ###
+  database:
```
Comment (weaverryan) [inline docker-compose.yaml:15]: Good job using this. But now delete the database stuff above this :)

**Review by weaverryan — COMMENTED** [review]
Yes, very well done! This container all the recipe updates I expected. Did you use the unreleased recipes:update? Or are you just really good at using the old recipe update system?

File: composer.json (hunk 1 above)
Comment (bocharsky-bw) [reply composer.json:12]: I didn't... that bundle already has Sf6-related PR, but that one is a bit more complex than just composer constraints

**Comment by bocharsky-bw** [comment]
> Did you use the unreleased recipes:update? Or are you just really good at using the old recipe update system?

Ha, I didn't know about the `recipes:update`, I just did that simple`recipes:install --force` :)

File: docker-compose.yaml (hunk 2 above)
Comment (bocharsky-bw) [reply docker-compose.yaml:5]: whoops, missed :)

**Pull request merged**